src/timeseriesflattener/testing/utils_for_testing.py

```diff
@@ -117,6 +117,23 @@ def assert_flattened_data_as_expected(
         raise ValueError("Must provide an expected set of data")
 
 
+def create_long_df():
```

Comment:
Use load instead of create for consistency
```suggestion
def load_long_df_with_multiple_values():
```